src/Server/Server.fs

```diff
@@ -94,10 +108,25 @@ let testWebApp =
     |> Remoting.buildHttpHandler
 
 
+let annotatorDocs = Docs.createFor<IAnnotatorAPI>()
```

Comment:
Does this work well? if so, we should expand it and provide an api doc link in the repo and on the page

Reply:
This works really well if you use examples. Otherwise the documentation lacks in comparison to OpenApi plus you cannot choose which Apis to document. The documentation for all api is automatic and you only need to fill in description and examples.